development/tools/jobs/kyma/apiserver_proxy_test.go

```diff
@@ -0,0 +1,62 @@
+package kyma_test
+
+import (
+	"testing"
+
+	"github.com/kyma-project/test-infra/development/tools/jobs/tester"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+)
+
+func TestApiServerProxyJobsPresubmit(t *testing.T) {
+	// WHEN
+	jobConfig, err := tester.ReadJobConfig("./../../../../prow/jobs/kyma/components/apiserver-proxy/apiserver-proxy.yaml")
+	// THEN
+	require.NoError(t, err)
+
+	assert.Len(t, jobConfig.Presubmits, 1)
+	kymaPresubmits, ex := jobConfig.Presubmits["kyma-project/kyma"]
+	assert.True(t, ex)
+	assert.Len(t, kymaPresubmits, 1)
+
+	actualPresubmit := kymaPresubmits[0]
+	expName := "kyma-components-apiserver-proxy"
+	assert.Equal(t, expName, actualPresubmit.Name)
+	assert.Equal(t, []string{"master"}, actualPresubmit.Branches)
+	assert.True(t, actualPresubmit.SkipReport)
+	tester.AssertThatJobRunIfChanged(t, actualPresubmit, "components/apiserver-proxy/")
```

Comment:
Please add this assertation as well: `assert.Equal(t, "^components/apiserver-proxy/", actualPresubmit.RunIfChanged)`

Reply:
Done.